backend/climateconnect_main/urls.py

```diff
@@ -14,23 +14,23 @@
     2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
 """
 from django.contrib import admin
-from django.urls import path
+from django.urls import path, include
 from climateconnect_api.views import (
     status_views, user_views
 )
 from knox import views as knox_views
 from django.conf import settings
 from django.conf.urls.static import static
 
-
-
-
 urlpatterns = [
     path('admin/', admin.site.urls),
     path('ping/', status_views.PingPongView.as_view(), name='ping-pong-api'),
     path('login/', user_views.LoginView.as_view(), name='login-api'),
     path('logout/', knox_views.LogoutView.as_view(), name='logout-api'),
     path('signup/', user_views.SignUpView.as_view(), name="signup-api"),
     path('api/my_profile/', user_views.PersonalProfileView.as_view(), name='user-profile-api'),
-    path('api/members/', user_views.MemberProfilesView.as_view(), name="member-profiles-api")
-]+ static(settings.MEDIA_URL, document_root=settings.MEDIA_ROOT)
+    path('api/members/', user_views.MemberProfilesView.as_view(), name="member-profiles-api"),
+    path('api/', include('organization.urls')),
```

Comment:
Does this mean if the organization's name was 'greenpeace' the url would be `/api/greenpeace`? I think it would make most sense for it to be `/api/organizations/greenpeace`.

Reply:
No this will includes urls.py file in organization directory. Its a way to include URLs in other files. Its common approach taken in Django for better API URL management.